dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/program/notification/ProgramNotificationTemplateServiceTest.java

```diff
@@ -79,11 +81,11 @@ void setUp() {
     organisationUnitService.addOrganisationUnit(organisationUnit);
     program = createProgram('P');
     program.setAutoFields();
-    program.setUid("P_UID_1");
+    program.setUid(CodeGenerator.generateUid());
```

Comment:
If you don't need to set the UID you can remove this line.
`createProgram` already assign a UID to it.